samples/subsys/tracing/sample.yaml

```diff
@@ -2,16 +2,19 @@ common:
   tags:
     - tracing
     - debug
-  harness: console
+    - gpio
+  depends_on: gpio
+  harness: ztest
```

Comment:
The original testcase tests the tracing in a generic way, this changed that behavior and probably isn't ideal. You probably want to do this (gpio-tracing related) in the `sample.tracing.gpio`, or create another testcase if that isn't possible, and leave the other testcases alone